server/bin/www

```diff
@@ -51,7 +51,7 @@ const authSocket = async (socket, next) => {
   return next();
 };
 
-io.use(authSocket);
+// io.use(authSocket);
```

Comment:
Remove?

Reply:
Can I remove it from my branch without any problems?

Reply:
The auth socket work hasn't been pushed in yet so no problem if you remove this

Reply:
We shouldn't even have this code yet in your branch 🤔

Reply:
Ok. I've just removed it. Thanks